lib/phoenix/endpoint.ex

```diff
@@ -657,68 +654,14 @@ defmodule Phoenix.Endpoint do
 
       @doc false
       def __sockets__, do: unquote(Macro.escape(sockets))
-
-      @doc false
-      def __handler__(%{path_info: path} = conn, opts), do: do_handler(path, conn, opts)
-      unquote(dispatches)
-      defp do_handler(_path, conn, opts), do: {:plug, conn, __MODULE__, opts}
```

Comment:
We don't need to worry about rolling up all the socket endpoints as special handlers any longer; they're now dispatched as Plugs directly within Endpoint's Plug pipeline (which is now based on Plug.Router and not Plug.Builder)